xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1398,6 +1410,24 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
                     gf_upcall_list[i], fop_hits, "0", "0", "0");
     }
 
+    /* To measure FOP_HITS_PER_SEC consider time difference
+       between last profile stat started and last fop hit.
+       In case if last_fop_hit value is 0 then consider now
+       to measure the duration
+    */
+    if (conf->last_fop_hit)
+        duration = (conf->last_fop_hit - stats->started_at);
+    else
+        duration = (now - stats->started_at);
+
+    if (duration > 0) {
+        ios_log(this, logfp, "FOPS/S : %" PRIu64, (total_fop_hits / duration));
+        ios_log(this, logfp, "\n");
```

Comment:
nit: this can be moved after the `if`, since it's common code in both branches of the `if`.

Reply:
done